l10n_it_ricevute_bancarie/__manifest__.py

```diff
@@ -9,7 +9,7 @@
 
 {
     'name': "ITA - Ricevute bancarie",
-    'version': "12.0.1.0.2",
+    'version': "12.0.1.0.3",
```

Comment:
Questo può essere anche fatto dal maintainer che mergia la PR usando i comandi di ocabot (https://github.com/OCA/oca-github-bot/#commands).

Anzi, in generale meglio lasciare al maintainer la decisione di come incrementare al versione.